README.md

```
@@ -144,8 +146,16 @@ specification, compatible with the
 | `cpn-pr-review`         | Review console PRs: arch, French artifacts                                                            |
 | `cpn-pr-triage`         | Triage de PR : métadonnées, reviewers, lien issue                                                     |
 | `cpn-pr-workflow`       | Workflow PR : issue liée → PR draft → trier                                                           |
-| `cpn-stack`             | Isolation d'une unité en workspace jj frais : fork propre depuis main@origin, bookmark + push limités |
-| `cpn-swarm`             | Essaim d'agents A2A : routage par capacité, machine et pression runner                                |
+| `cpn-stack`             | Isolation d'une unité en workspace jj frais : fork propre depuis main@origin, bookmark + push limités | |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)